src/test/java/edu/kis/vh/nursery/RyhmersJUnitTest.java

```diff
@@ -61,19 +58,19 @@ public void testPeekaboo() {
 
 	@Test
 	public void testCountOut() {
-		IntArrayStack ryhmer = new IntArrayStack();
-		final int EMPTY_STACK_VALUE = -1;
+		DefaultCountingOutRyhmer ryhmer = new DefaultCountingOutRyhmer();
+		final int emptyStackValue = 0;

```

Comment:
Original constant should be used (it is better to use a constant from stacks interface).
Name could be UPPERCASE